src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    static final int COUNT_OF_FIGURES = 5;
```

Comment:
The constant 'COUNT_OF_FIGURES' should be private if it is not intended to be used outside of this class.

Reply:
fix the comment